lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -1510,22 +1510,22 @@ public function OrderByItem()
         $glimpse = $this->lexer->glimpse();
 
         switch (true) {
-            case ($this->isFunction()):
-                $expr = $this->FunctionDeclaration();
-                break;
-
-            case ($this->isMathOperator($peek)):
+            case $this->isMathOperator($peek):
```

Comment:
This and the change below should be one separate commit, since they have nothing to do with the issue at hand.

Reply:
The coding standard build fails if I just move down the list the `case ($this->isFunction()):` because of the now useless parenthesis.

So for consistency in the switch block I have also removed the parentheses of the others cases. I will send a fix without altering the other cases. Thanks for looking into it!

Reply:
You can add another commit in the same PR, that's ok!

Reply:
Ah in fact, forget about this: we have temporarily enabled a squash-only policy on this repository, I forgot about it. You can keep it in one single commit.